spec/test_runner_spec.rb

```diff
@@ -4,124 +4,228 @@
 
 describe ::Albacore::TestRunner::Config do
   it do
-    should respond_to :opts
```

Comment:
Because the other specs cover all these cases. We use `opts`, `files=` and the rest. I removed these tests largely due to the problems I've seen while working with tests that overly-specify the production code's behaviour, making it harder to refactor. Given your note here and [above](https://github.com/Albacore/albacore/pull/213/files/2ba65ce06c4f63b68661354041a175715e2858ce#r70756770), I now understand that the Config and Cmd classes are expected to do more than blindly serve the Task class. I will restore these. And add a `respond_to` for `execute_as_batch`.